packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockAudioMessageJsonFrom..st

```diff
@@ -0,0 +1,17 @@
+message - json
+mockAudioMessageJsonFrom: aChatId
+
+	^ (JsonObject newFrom: {
+			'@type' -> 'message'.
+			'id' -> self mockMessageID.
+			'chat_id' -> aChatId.
+			'sender' -> (JsonObject newFrom: {
+				'@type' -> 'messageSenderUser'.
+				'user_id' -> self mockUser1 id
+			}).
+			'content' -> (JsonObject newFrom: {
+				'@type' -> 'messageAudio'.
```

Comment:
what happens if we choose '''@type -> 'alwaysUnsupported' '''

Reply:
implemented your suggestion